### PR TITLE
BaSP-MR-167: automation test of Trainer from Admin and Sign Up of Members 

### PR DESCRIPTION
**Pull request by VictoriaNegrotto**
automation test - e2e about trainers from admin: post, edit and delete and e2e about sign up of member.

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run lint
+#npm run lint
```
Comment (Micaela-Rossi) [inline .husky/pre-commit:4]: why this change?

File: test/specs/member-e2e.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+     "Fisherton", "19-05-1995", "2000", "vicram2514");
+    await SignUpMember.optionSelectBlackClick();
+    await browser.pause(2000);
+    await Buttons.submitBtnClick();
+    await expect(ModalAlert.modalAlertText).toBeDisplayed();
+    await ModalAlert.modalAlertButtonClick();
+    await expect(browser).toHaveUrl('https://juvi-megarocket-app.vercel.app/auth/sign-in');
+  });
+
+  it('login member', async ()=>{
```
Comment (Micaela-Rossi) [inline test/specs/member-e2e.js:27]: this would be replaced for the login process when merge, don't forget!

File: test/specs/member-e2e.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    await LogIn.signInBtnClick();
+    await SignUpMember.signIn("pablomorad@hotmail.com", "Chimpance1");
+    await LogIn.submitBtnClick();
+    await expect (ModalAlert.modalAlertText).toBeDisplayed();
+    await ModalAlert.modalAlertButtonClick();
+    await expect(browser).toHaveUrl('https://juvi-megarocket-app.vercel.app/member');
+  });
+
+  it('navigate on activities', async ()=>{
```
Comment (Micaela-Rossi) [inline test/specs/member-e2e.js:36]: i don't know why test this, if the test of activities, it's done in this file in master

**Review by Micaela-Rossi — CHANGES_REQUESTED** [review]
Great Job! just some changes and warnings to not forget when merge

**Review by Micaela-Rossi — APPROVED** [review]
The changes seems good!

**Review by nicosimo8 — APPROVED** [review]
Solve conflicts and let's goooo
